staging_service/autodetect/GenerateMappings.py

```diff
@@ -264,12 +266,12 @@
     apps = mapping[category]
     for app in apps:
         # print("looking at", app)
-        title = app["title"].replace(" ", "_").lower()
+        title = app["title"]
 
         if title not in new_apps:
             # Create a new entry for extensions and id in the app
             app["extensions"] = []
-            app["id"] = counter
+            app["_id"] = counter
```

Comment:
Is there a reason not to use the new string `id` field here, and just do away with the new numerical `_id` field?

Reply:
Doing away with the _id field should probably be in the same PR as doing away with the "apps" field. That part of the API response is not needed anymore, because it there is a config already in the narrative which has both importer info, and cell metadata info, so its more coupled there anyway.